core/src/Network/GRPC/Unsafe.chs

```diff
@@ -139,11 +142,18 @@ castPeek p = do
 
 {#fun grpc_shutdown as ^ {} -> `()'#}
 
+{#fun grpc_shutdown_blocking as ^ {} -> `()'#}
```

Comment:
All the other language bindings seem to have switched to that so it seems sensible to switch `grpc-haskell` as well.